### PR TITLE
Fix generator bugs

### PR DESCRIPTION
**Pull request by alloy**
See commits for details.

Closes https://artsyproduct.atlassian.net/browse/PLATFORM-1411

### CODE REVIEW

File: packages/cli/src/generator/templates/Component.tsx.ejs [hunk 1]
```diff
@@ -25,26 +25,28 @@ export const <%= componentName %>: React.SFC<<%= componentName %>Props> = props
...
+            }
           }
         }
-      }
 <% } -%>
-    }
-  `,
+      }
+    `,
+  },
```
Comment (ds300) [inline packages/cli/src/generator/templates/Component.tsx.ejs]: I can't figure out where (these template files are :scream:), but we're missing a closing brace on fragment containers (pagination containers are fine)

image

Comment (alloy) [reply packages/cli/src/generator/templates/Component.tsx.ejs]: Oh wow, great catch!

Comment (alloy) [reply packages/cli/src/generator/templates/Component.tsx.ejs]: I guess the test should ideally try to parse the GraphQL documents to ensure they aren't broken in these ways

Comment (ds300) [reply packages/cli/src/generator/templates/Component.tsx.ejs]: great idea :+1:

Comment (alloy) [reply packages/cli/src/generator/templates/Component.tsx.ejs]: Pretty happy with this:

Screenshot 2019-05-20 at 18 52 35

**Comment by alloy** [comment]
Ok, fixed that issue, going to merge and release 👍

File: packages/cli/src/generator/templates/Component.tsx.ejs (hunk 1 above)
Comment (ds300) [reply packages/cli/src/generator/templates/Component.tsx.ejs]: Yessss thank you :pray:

**Pull request merged**